sass/components/_tooltips.scss

```diff
@@ -0,0 +1,112 @@
+// subset of hint.css
+// scss-lint:disable ImportantRule,MergeableSelector,PropertySortOrder
+// scss-lint:disable PseudoElement,VendorPrefix
+
+[class*="hint--"] {
```

Comment:
this was just a carry-over from their css file (didn't want to just use as is because it has a lot of stuff we don't need)